src/machine/machine_mimxrt1062.go

```diff
@@ -0,0 +1,591 @@
+// +build mimxrt1062
+
+package machine
+
+import (
+	"device/nxp"
+	"runtime/volatile"
+)
+
+// Peripheral abstraction layer for the MIMXRT1062
+
+func CPUFrequency() uint32 {
+	return 600000000
+}
+
+type PinMode uint8
+
+// muxSelect is yet another level of indirection required to connect pins in an
+// alternate function state to a desired peripheral (since more than one pin can
+// provide a given alternate function).
+//
+// Once a pin is configured with a given alternate function mode, the IOMUXC
+// device must then be configured to select which alternate function pin to
+// route to the desired peripheral.
+//
+// The reference manual refers to this functionality as a "Daisy Chain". The
+// associated docs are found in the i.MX RT1060 Processor Reference Manual:
+//   "Chapter 11.3.3 Daisy chain - multi pads driving same module input pin"
+type muxSelect struct {
+	mux uint8                // AF mux selection (NOT a Pin type)
+	sel *volatile.Register32 // AF selection register
+}
+
+// connect configures the IOMUXC controller to route a given pin with alternate
+// function to a desired peripheral (see godoc comments on type muxSelect).
+func (s muxSelect) connect() {
+	s.sel.Set(uint32(s.mux))
+}
```

Comment:
This code appears to be dead.

Reply:
Dead code for this PR, since GPIO is the default mux selection for all pins, but this is very much needed in all subsequent peripheral PRs.